packages/styles/settings-tools/_u.preview-grid.scss

```diff
@@ -0,0 +1,33 @@
+$grid-line-color: rgba(0, 100, 100, 0.1);
```

Comment:
add documentation for it in the `magic-unit` section

Reply:
fixed